<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="sortComboBox">
      <item>
       <property name="text">
        <string>A-Z</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Z-A</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="addProjectButton">
      <property name="text">
       <string>Dodaj Projekt</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTableWidget" name="projectTableWidget"/>
    </item>
    <item row="0" column="3">
     <widget class="QWidget" name="widget_3" native="true"/>
    </item>
    <item row="0" column="4">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="sortButton">
      <property name="text">
       <string>Sortuj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
